src/doodle/doodleCanvas.js

```diff
@@ -34,6 +34,11 @@ const DoodleCanvas = ({
 }) => {
   const [isDrawing, setIsDrawing] = useState(false);
 
+  useEffect(() => {
+    //@ts-ignore
+    document.getElementById('canvas').className = 'size' + size;
```

Comment:
You can set the className directly in this file, in the `<Canvas />` return below.